ledger/streaming/client/src/grpc.rs

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+use displaydoc::Display;
+use futures::{Stream, StreamExt};
+use mc_common::logger::{log, o, Logger};
+use mc_crypto_keys::Ed25519Public;
+use mc_ledger_streaming_api::{
+    parse_subscribe_response,
+    streaming_blocks::{SubscribeRequest, SubscribeResponse},
+    streaming_blocks_grpc::LedgerUpdatesClient,
+    BlockStream, BlockStreamComponents, Result,
+};
+use mc_util_grpc::ConnectionUriGrpcioChannel;
+use mc_util_uri::ConnectionUri;
+use std::{sync::Arc, time::Duration};
+
+/// A BlockStream that streams blocks using the `LedgerUpdates` gRPC API.
+#[derive(Display)]
+pub struct GrpcBlockSource {
```

Comment:
This is a rename of `source.rs` with minor modifications, enough to throw off git's rename detection.